src/platform.ts

```diff
@@ -469,6 +475,17 @@ export function getWindowsSystemPowerShellPath(systemFolderName: string) {
         "powershell.exe");
 }
 
+function fileExistsSync(filePath: string): boolean {
+    try {
+        // This will throw if the path does not exist,
+        // and otherwise returns a value that we don't care about
+        fs.lstatSync(filePath);
```

Comment:
This probably ought to just be `fs.statSync` since we don't care about the information, but we _may_ care that the target file exists. I don't really care though, as the code as written is (presumably) tested, and it's not really our problem if the symlink exists but its target does not.